### PR TITLE
start: Data Access and Data Versioning to mention Model in titles (#2096)

### LINKED ISSUE #2096: start: Data Access mentions tracking models without clarifying what kind
I was reading the documentation on [Data Access](https://github.com/iterative/dvc.org/blob/master/content/docs/start/data-access.md). It starts by mentioning that 
```quote
Okay, now that we've learned how to track data and models with DVC and how to version them with Git, next question is how can we use these artifacts outside of the project?
```
However, the previous tutorial [Get Started: Data Versioning](https://github.com/iterative/dvc.org/blob/master/content/docs/start/data-versioning.md) only shows how to track data and tracking models are not demonstrated.

### PR DESCRIPTION
**Pull request by iesahin**
UPDATE: Jump to https://github.com/iterative/dvc.org/pull/2214#pullrequestreview-622871139

> Added changes from PR #2188 and modified paths & titles

Fixes #2096

- Changes the title of "Data Access" to "Data and Model Access"
- Changes the title of "Data Versioning" to "Data and Model Versioning"
- Renames path of Data Access and Data Versioning to
  `data-and-model-access.md` and `data-and-model-versioning.md`
  respectively.
- Adds redirects
-- `/doc/start/data-access` -> `/doc/start/data-and-model-access`
-- `/doc/start/data-versioning` ->
`/doc/start/data-and-model-versioning`
- Replaces links in `/doc/start` with the new links.

### TODO

- [x] Replace all links to `data-access` and `data-versioning` with the new paths. (DONE in bb84a996042c3abfa0e039cfbc96392bdc1f6617 9ef97c6c1fa4076e818f1213fd3aa8644df0b952 2593bb7923715b43802acccdaf82227faa075e6a 9ed0867cb7eb7572eba2036e77955c7e9ec2c74a 3d7d61defce67be42121c5788fe8f7d93d8fd43f )

### CODE REVIEW

**Comment by iesahin** [comment]
I can replace all links to `/doc/start/data-access` and `/doc/start/data-versioning` with the new links. Is this OK?

**Comment by shcheklein** [comment]
@iesahin looks good to me! let me know when it's ready to be merged, I see that you still have a TODO?

**Comment by shcheklein** [comment]
We can also probably add `Fixes #2096` to the description?

File: redirects-list.json
Comment (jorgeorpinel) [inline redirects-list.json:30]:
```suggestion
  "^/doc/start/data-versioning$                                                           /doc/start/data-and-model-versioning",
  "^/doc/start/data-access$                                                               /doc/start/data-and-model-access",
```

File: content/docs/start/data-and-model-versioning.md
Comment (jorgeorpinel) [inline content/docs/start/data-and-model-versioning.md:258]: Interesting. What's the purpose of this section?

Maybe we can just make sure we mention models enough in the existing text e.g. early on near "To start tracking a file or directory" we could say ", whether its raw data or a trained ML model"

I'm not opposed to also adding the section but it's unclear to me as of now what added value it has i.e. what is special about versioning model files vs. any large file?

File: content/docs/start/data-and-model-access.md
Comment (jorgeorpinel) [inline content/docs/start/data-and-model-access.md:8]: > Okay, now that we've ... with Git.

Incomplete sentence

File: content/docs/start/data-and-model-versioning.md
Comment (shcheklein) [reply content/docs/start/data-and-model-versioning.md:258]: My bet that @iesahin was trying to make it extra explicit. My 2cs - I'm fine to keep it, be 100% that we have all parts covered now :)

File: content/docs/start/data-and-model-access.md
Comment (jorgeorpinel) [inline content/docs/start/data-and-model-access.md:9]: Maybe "Machine learning models typically are also large files in the project. DVC can ..."

File: content/docs/start/data-and-model-access.md
Comment (jorgeorpinel) [inline content/docs/start/data-and-model-access.md:11]: Would be ideal to keep a single paragraph here.

Comment (jorgeorpinel) [reply content/docs/start/data-and-model-access.md:11]: Also, "The next questions are ..."

**Review by jorgeorpinel — CHANGES_REQUESTED** [review]
Getting there!

File: content/docs/start/data-and-model-versioning.md
Comment (jorgeorpinel) [reply content/docs/start/data-and-model-versioning.md:258]: I'm not opposed to keeping it but the text is a bit repetitive, I'd like to read what makes versioning model files special. I don't have a more specific suggestion for now...

Comment (iesahin) [reply content/docs/start/data-and-model-versioning.md:258]: Actually the gist is _models are also files and tracked as files_ and normally I wouldn't consider mentioning this explicitly. However it looks we need to be more explicit here. I think _model_ seems a rather abstract concept in user's mind and _you can keep track of your models_ as a separate section seems a better way for handling for #2096.

File: content/docs/start/data-and-model-access.md
Comment (iesahin) [reply content/docs/start/data-and-model-access.md:9]: Should this also mention "Data Science"? I'm also thinking about SEO.

File: redirects-list.json
Comment (iesahin) [reply redirects-list.json:30]: How does this work for paths like `/doc/start/data-versioning#subsection`? (Just asking to learn) 

I think it doesn't pass `#subsection` to the redirected page in my version, right?

File: content/docs/start/data-and-model-access.md
Comment (iesahin) [reply content/docs/start/data-and-model-access.md]: Actually _version_ is a verb there. Fixed in 09dc8ca7c498ee0414c3b6bce7430cc82054f012. Thanks.

File: content/docs/start/data-and-model-access.md
Comment (iesahin) [reply content/docs/start/data-and-model-access.md]: Fixed in a3b15ba4e4a463fe47884150fc25238b85853cbe. Thanks. 👍🏼

File: content/docs/start/data-and-model-access.md
Comment (iesahin) [reply content/docs/start/data-and-model-access.md]: Revised in 7731587eb58c58805f5ec7c5f398dc385b1dce9c.

**Comment by iesahin** [comment]
Links in blogs to new paths (in the form of `dvc.org/doc/start/...`) fails. I can keep them, revert them to let redirection do its work, or change them to `/doc/start/` links. @jorgeorpinel @shcheklein

File: yarn.lock
Comment (shcheklein) [inline yarn.lock:13381]: @iesahin please, make a separate PR for this change so that we fix master

File: redirects-list.json
Comment (jorgeorpinel) [inline redirects-list.json:31]: Can we insert these in line 25 instead? Right after the `/doc/get-started` redirect please. No blank lines needed. Thanks

File: content/docs/start/data-and-model-versioning.md
Comment (jorgeorpinel) [inline content/docs/start/data-and-model-versioning.md:252]: I don't think "eases" means "makes easy". Let's try to use simple straightforward language like:

```suggestion
DVC helps you handle model files too. Models in a
```

Comment (jorgeorpinel) [reply content/docs/start/data-and-model-versioning.md:252]: Although maybe we don't need this sentence? (See https://github.com/iterative/dvc.org/pull/2214#pullrequestreview-593674216). Thanks

File: content/docs/start/data-and-model-versioning.md
Comment (jorgeorpinel) [reply content/docs/start/data-and-model-versioning.md:258]: > the gist is models are also files and tracked as files and normally

Perfect, let's focus on that 🙂. Preferably as the opening sentence: "Model files are no different than data files when it comes to tracking their versions."

**Then** add the model-specific info like "models change more frequently than data".

Comment (jorgeorpinel) [reply content/docs/start/data-and-model-versioning.md:258]: Other comments:

> provides means to track minor changes

What are we referring to here?

> In later sections of this series, you'll see how DVC enables to track changes in pipelines

The comment makes sense in general but probably not for this **Model versioning** section. Let's just focus on the task and skip it, I think.

Thanks

File: content/docs/start/data-and-model-access.md
Comment (jorgeorpinel) [reply content/docs/start/data-and-model-access.md]: Still incomplete. "to version" was OK. The part that is messing the sentence up is "now that". Please either remove that or complete the sentence after "in Git, (something)" (or rewrite in some other complete way). Thanks

File: yarn.lock
Comment (iesahin) [reply yarn.lock:13381]: I think it's OK now.

Comment (shcheklein) [reply yarn.lock:13381]: yep, thanks! can we merge/rebase so that we don't see it here, please?

File: content/docs/start/data-and-model-access.md
Comment (jorgeorpinel) [reply content/docs/start/data-and-model-access.md]: > Should this also mention "Data Science"? I'm also thinking about SEO.

Was it in the desired keywords? Someone actually did research to come up with those. I see they're gone now (prob. should've left that comment until right before merging this)

Comment (jorgeorpinel) [reply content/docs/start/data-and-model-access.md]: I see this still reads "Models in a machine learning project". Let's use "Machine learning models" please (SEO).

File: content/docs/start/data-and-model-access.md
Comment (jorgeorpinel) [reply content/docs/start/data-and-model-access.md]: Thanks. But it's still missing "The".

File: content/docs/start/data-and-model-versioning.md
Comment (iesahin) [reply content/docs/start/data-and-model-versioning.md]: > What are we referring to here?

I'm thinking about run-cache, parameters etc. Features that work separately from Git. These are related to models rather than data.

File: content/docs/start/data-and-model-access.md
Comment (iesahin) [reply content/docs/start/data-and-model-access.md]: Keywords are in BC documents, this is a GS document. There were no keywords here 😄

File: content/docs/start/data-and-model-access.md
Comment (iesahin) [reply content/docs/start/data-and-model-access.md]: Fixed in b46cca31

File: content/docs/start/data-and-model-access.md
Comment (iesahin) [reply content/docs/start/data-and-model-access.md]: Removed `Okay, now that` in b46cca31

File: content/docs/start/data-and-model-versioning.md
Comment (iesahin) [reply content/docs/start/data-and-model-versioning.md]: I updated and removed `pipelines` in b46cca31 .

File: redirects-list.json
Comment (iesahin) [reply redirects-list.json]: Done in b46cca31. Thanks.

File: content/docs/start/data-and-model-versioning.md
Comment (jorgeorpinel) [reply content/docs/start/data-and-model-versioning.md]: Hey @iesahin sorry, this one fell through the cracks during the 2.0 release... Checking now. May take over if you're no longer available for this.

File: content/docs/start/data-and-model-versioning.md
Comment (jorgeorpinel) [inline content/docs/start/data-and-model-versioning.md:258]: > Continuing https://github.com/iterative/dvc.org/pull/2214#pullrequestreview-593674216

I'm still not convinced we need this new section. We already say "data and models" in every section (except in **Retrieving** — let's fix that though), so if the gist here is that models are also tracked as any file normally, I think that's already implied in every other section.

Also, it can probably be summarized a bit (see feedback below) and then it's too short for a whole section anyway (could be moved to right before **Storing and sharing** if anything).

- "usually change more frequently than data files" contradicts "are no different than data files" (at first sight)
- "provides means to track minor changes in model files" - vague, what do we mean? run-cache, parameters etc. are not specifically for models.
- The last sentence seems unnecessary.

File: content/docs/start/data-and-model-access.md
Comment (jorgeorpinel) [reply content/docs/start/data-and-model-access.md]: > Keywords are in BC documents

True haha, nvmd.

File: content/docs/start/data-and-model-access.md
Comment (jorgeorpinel) [inline content/docs/start/data-and-model-access.md:12]: This change seems to break the semantics of the paragraph. TBH I don't think it's necessary, we already state "data and models".

I do like the correction to the first sentence, "... , and how to commit their versions to Git" but the next 2 sentences are out of context here.

File: content/docs/start/data-and-model-versioning.md
Comment (jorgeorpinel) [inline content/docs/start/data-and-model-versioning.md:259]:
```suggestion
```

File: content/docs/start/data-and-model-access.md
Comment (jorgeorpinel) [inline content/docs/start/data-and-model-access.md:12]:
```suggestion
Okay, we've learned how to _track_ data and models with DVC, and how to commit
their versions to Git. The next questions are: How can we _use_ these artifacts
outside of the project? How do I download a model to deploy it? How to download
a specific version of a model? Or reuse datasets across different projects?
```

**Review by jorgeorpinel — APPROVED** [review]
I left 2 still-pending questions above ☝️ 

In order to merge this (the title and link changes should be useful, thanks @iesahin), I'm basically rolling back those 2 things but feel free to continue discussing/explaining and open another PR if needed.

**Comment by jorgeorpinel** [comment]
@shcheklein do we still want to merge this? If so I'll solve the conflicts and do so. Thanks

**Comment by jorgeorpinel** [comment]
> do we still want to merge this?

p.s. I guess so, since #2096 is labeled `bug`. So I solved the conflicts. But will let you merge just to double check since there are new redirects here. Thanks

**Comment by jorgeorpinel** [comment]
p.p.s. everything here is link updates except content/docs/start/data-and-model-versioning.md and a bit more in content/docs/start/data-and-model-access.md

**Review by shcheklein — APPROVED** [review]

**Pull request merged**
